public-docs/intro-glance.md

```diff
@@ -0,0 +1,80 @@
+---
+title: Reaction at a Glance
+---
+
+Deciding whether Reaction is right for your company? Here's everything you need to know.
+
+## What is Reaction Commerce?
+
+Reaction Commerce is an open source commerce platform that allows anyone to customize, extend, and deploy an online store or marketplace. Our codebase is available for free via [GitHub](https://github.com/reactioncommerce/reaction).
+
+## Demo
+
+You can check out a live shop and use the admin tool in our [Demo Swag](https://swag.getreaction.io/) shop. You can also run the demo locally by cloning the [Reaction Swag Shop](https://github.com/reactioncommerce/reaction-swag-shop) repository.
```

Comment:
The swag shop code is going to be deprecated as of 2.0 in favor of the new NextJS-focused tutorial.

Reply:
What do you suggest it should say in this section?

Reply:
Should we just say you can run Reaction locally? If Heroku is still working we could also point them to that although it's plain vanilla Reaction not swag shop. (we should really be loading the sample data that Erik created by default)

Reply:
But wouldn't you still go to the current swag shop URL to play with admin features? Maybe we just need to be more specific that that's the admin UI and there's a newer better storefront UI.

Reply:
And I think we should host the customized NextJS swag UI, too. That one is simple since it wouldn't ever need to refresh.

Reply:
Yeah, I put in a ticket for Grig to figure out how to host it. Maybe a version that could point to the swag shop admin so you could see your changes reflected in the starter kit. it's probably worth talking to Cindy about as she relies on this stuff for sales.

It's just a little bit more of a complicated story than 'go here to see reaction" now. And we probably need to start thinking of these parts of the docs as "sales" documents

Reply:
I just had a discussion with Grig about it and he is putting together a plan. For now you can just point at swag.getreaction.io and be done with it and I will update the docs more once I know what the plan is

Reply:
Thanks. I too was thinking that this stuff should be on a marketing website, but for now I at least wanted to surface it out of the FAQs and collect in one article.